### ISSUE TITLE
RSS article is not updated as expected

### ISSUE THREAD

**Issue by elbidone** [meumobi/sitebuilder#311]
Category 475 from C-SE env (site_id 61) use extension RSS (http://www.raiadrogasil.com.br/FeedMeuMobi.aspx?idCanal=2nlFyAThvxbpyfXPHKc5zw==)
The item "4T15" have been updated by the client but not updated on meumobi as expected.
See below current content, only one media appears as published originally by the client:
![raia_drogasil_pt](https://cloud.githubusercontent.com/assets/9011612/13289864/49d8b80e-daf0-11e5-90cc-457280eaa2c8.png)

And here the current feed:
```xml
<item>
	<title>4T15</title>
	<media:content url="http://www.raiadrogasil.com.br/Download.aspx?Arquivo=6XnDzF8ybKBMk77PaXTy7w==" fileSize="1300" type="application/pdf" medium="document">
		<media:title>Resultados 2015</media:title>
	</media:content>
	<media:content url="http://www.raiadrogasil.com.br/Download.aspx?Arquivo=BuzOcwxKXQomnhSls1F8NA==" fileSize="1580" type="application/pdf" medium="document">
		<media:title>DFP 2015</media:title>
	</media:content>
	<media:content url="http://www.raiadrogasil.com.br/Download.aspx?Arquivo=hWqN6IB7YVCAs9ZF9c+76w==" fileSize="1229" type="application/pdf" medium="document">
		<media:title>Apresenta&#xE7;&#xE3;o 2015</media:title>
	</media:content>
	<media:content url="http://www.raiadrogasil.com.br/Download.aspx?Arquivo=VtGO9Sw07vBxBskOYuIx4Q==" fileSize="14513" type="audio/mpeg" medium="audio">
		<media:title>&#xC1;udio 2015</media:title>
	</media:content>
	<media:content url="http://www.raiadrogasil.com.br/Download.aspx?Arquivo=flDQ+xMMRXcaKpcD5GYGiQ==" fileSize="5982" type="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" medium="document">
		<media:title>Informa&#xE7;&#xF5;es 2014 e 2015</media:title>
	</media:content>
	<guid isPermaLink="false">
http://www.raiadrogasil.com.br/ShowResultado.aspx?IdResultado=+/OxE1swMWa6xBDF/NkLZA==
</guid>
	<link>
http://www.raiadrogasil.com.br/ShowResultado.aspx?IdResultado=+/OxE1swMWa6xBDF/NkLZA==
</link>
	<pubDate>Thu, 18 Feb 2016 00:00:00 GMT</pubDate>
</item>
```

**Comment by elbidone** [comment]
I've forced the update to provide up-to-date feed to client

**Comment by elbidone** [comment]
The description of the item is empty, the only changes between original post and updated are on media (news media added), so maybe the way we use the checksum doesn't allow to define if article have been update or not. We should include media on checksum. 
Please detail on wiki page how we manage updates of files, which fields are considered on checksum.

**Comment by juliogreff** [comment]
Tests performed:

* Create a category with an RSS extension pointing to [a feed](http://www.raiadrogasil.com.br/FeedMeuMobi.aspx?idCanal=2nlFyAThvxbpyfXPHKc5zw==).

* Run the UpdateFeeds worker

```
$ php sitebuilder/script/run_worker.php --extension_id=5651ef213f482eac97d63af8 --worker=UpdateFeeds
```

* By now, all items should have media. To simulate an update, I deleted the media from one of the items.

```
> db.items.update({ _id: ObjectId("56dc5b413f482e444d0041c3") }, { $set: { medias: [] }});
```

* I run the UpdateFeeds worker again. Logs show that item was updated, and nothing else was touched. Database query shows that new media is present.

```
[2016-03-06 13:44:43] sitebuilder.INFO: starting script {"script_id":"UpdateFeeds-56dc5e7b50b11","component":"cli"} []
[2016-03-06 13:44:43] sitebuilder.INFO: fetching feed {"url":"http://www.raiadrogasil.com.br/FeedMeuMobi.aspx?idCanal=2nlFyAThvxbpyfXPHKc5zw==","extension_id":"5651ef213f482eac97d63af8","catego
ry_id":27,"component":"update_news_feed"} []
[2016-03-06 13:44:44] sitebuilder.INFO: feed fetched {"url":"http://www.raiadrogasil.com.br/FeedMeuMobi.aspx?idCanal=2nlFyAThvxbpyfXPHKc5zw==","extension_id":"5651ef213f482eac97d63af8","categor
y_id":27,"component":"update_news_feed"} []
[2016-03-06 13:44:44] sitebuilder.DEBUG: item will be updated {"item_id":"56dc5b413f482e444d0041c3","guid":"http://www.raiadrogasil.com.br/ShowResultado.aspx?IdResultado=mwGIo2fZ644sQkabBtgaKA=
=","category_id":27,"extension_id":"5651ef213f482eac97d63af8","component":"update_news_feed"} []
[2016-03-06 13:44:44] sitebuilder.INFO: item updated {"item_id":"56dc5b413f482e444d0041c3","site_id":4,"category_id":27,"component":"items"} []
[2016-03-06 13:44:44] sitebuilder.INFO: job created {"job_id":"56dc5e7c3f482e9a4e0041a7","type":"process_remote_media","params":{"item_id":"56dc5b413f482e444d0041c3"},"component":"worker_manage
r"} []
[2016-03-06 13:44:44] sitebuilder.INFO: script finished {"script_id":"UpdateFeeds-56dc5e7b50b11","component":"cli"} []
```

To test that the original code was not broken, I also deleted all items from that same category, and ran the UpdateFeeds worker again. All items were reimported successfully.

**Comment by juliogreff** [comment]
Closed in 68aafb72fc62e649199e7ac3dea657f40a498c6f.
